regulations/templates/regulations/preamble-chrome.html

```diff
@@ -53,11 +53,12 @@ <h3 class="toc-type">Table of Contents</h3>
 
     <div class="comment-wrapper" data-section="{{ sub_context.node.markup_id }}">
       <div class="toggle">(click to toggle comment box)</div>
-
-      <div class="comment">
+      <div class="comment" data-section="{{ sub_context.node.markup_id }}">
```

Comment:
I'm guessing this should only live on the .comment or the .comment-wrapper, but not both. Ultimately, I don't think we'll need either -- we should be able to pull the data from the marked up preamble/similar.